backend/src/app/migrations/sql/XXXX-trivial-spelling-fixes.sql

```diff
@@ -0,0 +1,4 @@
+alter index color_colection__team_id__idx to color_collection__team_id__idx;
+alter index icon_colection__team_id__idx to icon_collection__team_id__idx;
+alter index file__modified_at__has_media_trimed__idx to file__modified_at__has_media_trimmed__idx;
+alter index media_bject__file_id__idx rename to media_object__file_id__idx;
```

Comment:
Normally I make inline changes, but I think this is the right approach...

Reply:
:+1:

Reply:
we will need to rename it to correct number and include it to the migrations data structure, this is just a note for myself, it is ok as it is right now.